docs/config.md

```diff
@@ -15,6 +15,16 @@ with environment variables taking precedence over `App.config` or `Web.config` f
     file  (`web.config`) or an application configuration file (`app.config`) to
     configure the `OTEL_*` settings.
 
+    ⚠️ Only settings starting with `OTEL_` can be set using `App.config` or `Web.config`.
+    Additionally, following settings are not supported:
+
+    - `OTEL_DOTNET_AUTO_HOME`
+    - `OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES`
+    - `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE`
+    - `OTEL_DOTNET_AUTO_[TRACES|METRICS|LOGS]_[ENABLED|DISABLED]_INSTRUMENTATIONS`
+    - `OTEL_DOTNET_AUTO_LOG_DIRECTORY`
+    - `OTEL_DOTNET_AUTO_DEBUG`
```

Comment:
Might require some extra work, but I'm thinking now of re-doing the settings tables so that they have the following columns:

- Setting (env var)
- App.config / Web.config supported
- Description
- Default

Reply:
Can we keep as it is for this PR (it makes also reviewing easier)? We can change the formatting/structure it in a separate PR.